index.js

```diff
@@ -1701,69 +1702,74 @@ class GithubScm extends Scm {
      * Add a PR comment
      * @async  _addPrComment
      * @param  {Object}     config
-     * @param  {String}     config.comment     The PR comment
+     * @param  {Array}      config.comments    The PR comments
```

Comment:
need to update https://github.com/screwdriver-cd/data-schema/blob/master/plugins/scm.js#L107-L117